test/test_profiler.py

```diff
@@ -1,6 +1,9 @@
 import collections
 import gc
 import io
+import os
+import tempfile
```

Comment:
we've removed direct usage of tempfile earlier, because of an issue on windows
could you use the utility from torch/testing/_internal/common_utils.py (`TemporaryDirectoryName`)?

Reply:
Done, thanks.